pkg/controller/utils/namespace_helper.go

```diff
@@ -52,6 +57,9 @@ type NamespaceHelper interface {
 	// For single-tenant clusters, this simply returns the InstallNamespace.
 	TenantNamespaces(client.Client) ([]string, error)
 
+	// FilteredTenantNamespaces returns all namespaces for all Tenants that have Calico OSS managed clusters.
```

Comment:
I don't think this comment is accurate.

Reply:
Whoops, yeah that's from an earlier iteration.